packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataPersister/AbstractContextDataPersister.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\DataPersister;
+
+use ApiPlatform\Core\DataPersister\ContextAwareDataPersisterInterface;
+
+abstract class AbstractContextDataPersister implements ContextAwareDataPersisterInterface
+{
+    /**
+     * @var mixed
+     */
+    protected $dataPersister;
+
+    /**
+     * AbstractContextDataPersister constructor.
+     *
+     * @param mixed $dataPersister
+     */
+    public function __construct($dataPersister)
```

Comment:
The argument should be a `ContextAwareDataPersisterInterface` so no need for the PHPDoc and we should call it `$decorated`.